src/components/TodoList.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import { Todo } from '../types/Todo';
+import { TodoItem } from './TodoItem';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+}) => {
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+
+  return (
+    <>
+      {todos.map((todo) => (
+        <TodoItem
+          key={todo.id}
+          todo={todo}
+          selectedTodo={selectedTodo}
+          setSelectedTodo={setSelectedTodo}
+        />
+      ))}
+    </>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings